docker-compose.pipeline.yaml

```diff
@@ -0,0 +1,5 @@
+version: '3.4'
+services:
+  mine-support-calculation-service:
+    environment: 
+     MESSAGE_QUEUE_HOST : "amq"

```

Comment:
What's the reason for this being a separate docker-compose, rather than adding the environment variable to `docker-compose.test.yml`?

Reply:
I've merged the two and test works fine both locally and in the pipeline.  I've therefore removed the extra overhead.  Thanks.